snmp/datadog_checks/snmp/data/default_profiles/_cisco-ipsec-flow-monitor.yaml

```diff
@@ -0,0 +1,83 @@
+metrics:
+  - MIB: CISCO-IPSEC-FLOW-MONITOR-MIB
+    table:
+      name: cikeTunnelTable
+      OID: 1.3.6.1.4.1.9.9.171.1.2.3
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.15
+        name: cikeTunLifeTime
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.19
+        name: cikeTunInOctets
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.20
+        name: cikeTunInPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.21
+        name: cikeTunInDropPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.27
+        name: cikeTunOutOctets
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.28
+        name: cikeTunOutPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.29
+        name: cikeTunOutDropPkts
+      - OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.35
+        name: cikeTunStatus
+    metric_tags:
+      - tag: phase_1_tunnel_index
+        symbol:
+          OID: 1.3.6.1.4.1.9.9.171.1.2.3.1.1
```

Comment:
This OID is marked as `non-accessible`, I believe this means that we won't be able to get this OID with regular SNMP Get requests. I think we'll need to use an [index](https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#using-an-index)

I think the snmprec was not generated correctly, in particular I'm not expecting to see `1.3.6.1.4.1.9.9.171.1.2.3.1.1.*` in the output.

Reply:
ahhh ty for the pointer, totally missed that - i think my soon to come commit will fix it but i'll double check if i fully understood!

Reply:
hmmm also yea after regenerating the snmprec just in case there is still results showing `1.3.6.1.4.1.9.9.171.1.2.3.1.1.*` - i think there's always 2 - maybe i can just remove ? i'm not suer why it's doing that